webui/src/pages/repositories/repository/fileRenderers/simple.tsx

```diff
@@ -33,9 +33,9 @@ export const UnsupportedFileType: FC<RendererComponent> = ({path, fileExtension,
 }
 
 
-export const TextDownloader: FC<RendererComponentWithTextCallback> = ({ repoId, refId, path, onReady }) => {
+export const TextDownloader: FC<RendererComponentWithTextCallback> = ({ repoId, refId, path, usePresigned, onReady }) => {
```

Comment:
Where do we render TextDownloader with usePresigned?

Reply:
webui/src/pages/repositories/repository/fileRenderers/index.tsx